.travis.yml

```diff
@@ -3,6 +3,7 @@ language: python
 python:
   - 3.4.4
   - 3.5
+  - 3.4.2
```

Comment:
it would slow down all builds, unnecessarily.

We could add it to master builds.

Or I think we can also only run jobs on the nightly `cron`, which would reduce delays if multiple PRs are getting merged.